Makefile

```diff
@@ -23,9 +23,15 @@ push:
 e2e:
 	# parenthesis preserve current working directory
 	(cd testing/e2e && \
-	 go run ./main.go infra --subscription=${SUBSCRIPTION_ID} --tenant=${TENANT_ID} --names=${INFRA_NAMES} && \
+	 go run ./main.go infra --subscription=${SUBSCRIPTION_ID} --tenant=${TENANT_ID} --names=${INFRA_NAMES} --sp-app-obj-id=${SERVICE_PRINCIPAL_APP_OBJ_ID} && \
 	 go run ./main.go deploy)
 
+
+e2e-infra-only:
+	# parenthesis preserve current working directory
+	(cd testing/e2e && \
```

Comment:
this just does what make dev does right?

Reply:
this isn't a replacement for what make dev does. we have no tooling for using the infra created by this manually yet. 

we should remove this make target. keep make clean with straightforward targets that are useful.

This is basically mostly for if you want to test e2e infra changes you made yourself in which case you could just run `(cd testing/e2e && \
	 go run ./main.go infra --subscription=${SUBSCRIPTION_ID} --tenant=${TENANT_ID} --names=${INFRA_NAMES} --sp-app-obj-id=${SERVICE_PRINCIPAL_APP_OBJ_ID})`

Reply:
yeah i just used that for testing i'll take it out